resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/resources/session/AllSessionTests.java

```diff
@@ -25,7 +26,7 @@
 		TestBug12575.class, WorkspaceDescriptionTest.class, TestBug30015.class,
 		TestMasterTableCleanup.class,
 		ProjectPreferenceSessionTest.class, TestBug113943.class, TestCreateLinkedResourceInHiddenProject.class,
-		Bug_266907.class, TestBug297635.class, TestBug323833.class,
+		Bug_266907.class, Bug_297635.class, TestBug323833.class,
```

Comment:
You mentioned in the description that this test does **not** need to run as a session test anymore. That would mean that you can delete this and leave it only in the other suite.

Reply:
Fixed in 69978f2.